src/style.css

```diff
@@ -1,3 +1,67 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  display: flex;
+  padding: 0 50px;
+  align-items: center;
+}
+
+.image {
+  width: 40px;
+  height: 40px;
+  display: block;
+}
+
+nav {
+  display: block;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 60px;
+  text-transform: uppercase;
+}
+
+nav ul {
+  list-style: none;
+  display: flex;
+}
+
+nav ul li a {
+  display: flex;
+  text-decoration: none;
+  color: #000;
+  margin-right: 20px;
+}
+
+nav ul li:last-child a {
+  margin-right: 0;
+}
+
+nav ul li a.is-active {
+  color: #00acdc;
+}
+
+nav ul li a.is-active::after {
+  content: '';
+  position: absolute;
```

Comment:
The `::after` pseudo-element in the `.is-active` class uses `position: absolute;` but there is no `position: relative;` set on a parent element. This might cause the pseudo-element to be positioned incorrectly. Consider setting `position: relative;` on the parent element to ensure correct positioning.